README.md

```diff
@@ -0,0 +1,8 @@
+# Educational project from BrainRocket
+
+Here I will practice HTML and CSS layout skills
+
+## Content
+
+- [Technologies](#iHTML and CSS)
+- [License](#no)
```

Comment:
Add please info about student and mentor, like:
Student — Name Surname 
Mentor — Name Surname 

Add styles. 
Student and mentor should be italic.
Name Surname should be bold.

Reply:
OK. Done.